ext/mysqli/mysqli.stub.php

```diff
@@ -547,225 +547,225 @@ function mysqli_connect_errno(): int {}
+function mysqli_field_seek(mysqli_result $mysqli_result, int $index): bool {}
 
 function mysqli_field_tell(mysqli_result $mysqli_result): int {}
 
 function mysqli_free_result(mysqli_result $mysqli_result): void {}
 
 #if defined(MYSQLI_USE_MYSQLND)
-function mysqli_get_connection_stats(mysqli $mysqli_link): array {}
+function mysqli_get_connection_stats(mysqli $connection): array {}
 
 function mysqli_get_client_stats(): array {}
 #endif
 
-function mysqli_get_charset(mysqli $mysqli_link): ?object {}
+function mysqli_get_charset(mysqli $connection): ?object {}
 
-function mysqli_get_client_info(?mysqli $mysqli_link = null): ?string {}
+function mysqli_get_client_info(?mysqli $connection = null): ?string {}
 
 function mysqli_get_client_version(): int {}
 
 function mysqli_get_links_stats(): array {}
 
-function mysqli_get_host_info(mysqli $mysqli_link): string {}
+function mysqli_get_host_info(mysqli $connection): string {}
 
-function mysqli_get_proto_info(mysqli $mysqli_link): int {}
+function mysqli_get_proto_info(mysqli $connection): int {}
 
-function mysqli_get_server_info(mysqli $mysqli_link): string {}
+function mysqli_get_server_info(mysqli $connection): string {}
 
-function mysqli_get_server_version(mysqli $mysqli_link): int {}
+function mysqli_get_server_version(mysqli $connection): int {}
 
-function mysqli_get_warnings(mysqli $mysqli_link): mysqli_warning|false {}
+function mysqli_get_warnings(mysqli $connection): mysqli_warning|false {}
 
 function mysqli_init(): mysqli|false {}
 
-function mysqli_info(mysqli $mysqli_link): ?string {}
+function mysqli_info(mysqli $connection): ?string {}
 
-function mysqli_insert_id(mysqli $mysqli_link): int|string {}
+function mysqli_insert_id(mysqli $connection): int|string {}
 
-function mysqli_kill(mysqli $mysqli_link, int $connection_id): bool {}
+function mysqli_kill(mysqli $connection, int $connection_id): bool {}
 
-function mysqli_more_results(mysqli $mysqli_link): bool {}
+function mysqli_more_results(mysqli $connection): bool {}
 
-function mysqli_multi_query(mysqli $mysqli_link, string $query): bool {}
+function mysqli_multi_query(mysqli $connection, string $query): bool {}
 
-function mysqli_next_result(mysqli $mysqli_link): bool {}
+function mysqli_next_result(mysqli $connection): bool {}
 
-function mysqli_num_fields(mysqli_result $mysql_result): int {}
+function mysqli_num_fields(mysqli_result $result): int {}
 
 function mysqli_num_rows(mysqli_result $mysqli_result): int|string {}
 
 /** @param string|int $value */
-function mysqli_options(mysqli $mysqli_link, int $option, $value): bool {}
+function mysqli_options(mysqli $connection, int $option, $value): bool {}
 
-function mysqli_ping(mysqli $mysqli_link): bool {}
+function mysqli_ping(mysqli $connection): bool {}
 
 #if defined(MYSQLI_USE_MYSQLND)
 function mysqli_poll(?array &$read, ?array &$write, array &$error, int $sec, int $usec = 0): int|false {}
```

Comment:
I would prefer not to use abbreviations here. Other functions like `usleep()` or `stream_set_timeout()` also use the full names:

`$sec` -> `$seconds`
`$usec` -> `$microseconds`

It will also stop my spell checker from yelling at me when I use these parameters. :)